package/rattler-build/scripts/fix_macos_lib_paths.py

```diff
@@ -52,10 +52,10 @@ def change_reexport_dylib(file_path, reexport_dylib):
 
 def scan_directory(directory, recursive=False):
     if recursive:
-        print(f"Recursively scanning dir: {scan_path}")
+        print(f"Recursively scanning dir: {directory}")
     else:
         print(f"Scanning dir: {scan_path}")
```

Comment:
Is `scan_path` defined?

Reply:
Yes, in the script it's on line 15.

Reply:
It looks like a bug to me and should be changed to `directory`.

Though it currently is the same, it might not be in the future.

Reply:
in the else block it should be `directory` too I think